### DISCUSSION TITLE
email_auth Production Server

### DISCUSSION THREAD

**Discussion by SadekMaachou** [saran-surya/email_auth_node#27] [Q&A]
![npm](https://user-images.githubusercontent.com/96054288/147848504-35e86e3e-b7dc-485a-92a2-47ced0b3d1ba.PNG)
hello,
i am facing an issue while typing this command "npm i && npm link" and this is the error i get in the terminal =>

" C:\Users\HP\Downloads\esiway-rep-master\esiway-rep-master\Sadek-OTP-main\custom>npm i && npm link
'npm' n’est pas reconnu en tant que commande interne
ou externe, un programme exécutable ou un fichier de commandes. "

can you help me please ? i already installed it but i get the same problem

**Comment by saran-surya** [comment]
> ![npm](https://user-images.githubusercontent.com/96054288/147848504-35e86e3e-b7dc-485a-92a2-47ced0b3d1ba.PNG) hello, i am facing an issue while typing this command "npm i && npm link" and this is the error i get in the terminal =>
> 
> " C:\Users\HP\Downloads\esiway-rep-master\esiway-rep-master\Sadek-OTP-main\custom>npm i && npm link 'npm' n’est pas reconnu en tant que commande interne ou externe, un programme exécutable ou un fichier de commandes. "
> 
> can you help me please ? i already installed it but i get the same problem

Hi @sadekmc , can you please make sure that you have installed npm, and set the path in the env variables.


Thanks
